kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```
@@ -321,6 +319,20 @@ removeAllAnchorText=Remove All PIDs
 removeAllVisibleAnchorText=Remove All Visible PIDs
 snapshotDownloadSearchBoxPlaceholder=Filter PIDs by name...
 
+deviceSnapshotDownloadModalHint=Select the PIDs for the service configurations that will be downloaded locally.
+deviceWiregraphDownloadModalHint=A partial snapshot containing the current wiregraph configuration will be generated and downloaded.
+formatDownloadHint=The configurations can be exported either as JSON or XML format.
+
+deviceSnapshotDownloadModalDescription=Select the PIDs for the service configurations that will be downloaded locally.
+deviceSnapshotDownloadFormatHint=The configurations can be exported either as JSON or XML format.
+
+deviceSnapshotRollbackConfirm=Choose the PIDs for which you want to perform the configuration rollback. During the rollback operation, the device may disconnect and reconnect, depending on which services are updated.
```

```suggestion
deviceSnapshotRollbackConfirm=Select the PIDs for which you wish to execute the configuration rollback. During this operation, the device may experience temporary disconnection and reconnection, depending on the updated services
```

Maybe an alternative rewrite